src/db/results.rs

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Result types for database methods.
+
+use std::collections::HashMap;
+
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+pub type GetCollections = HashMap<String, u64>;
+pub type GetCollectionCounts = HashMap<String, u64>;
+pub type GetCollectionUsage = HashMap<String, u32>;
+pub type GetQuota = Vec<u32>;
+pub type DeleteAll = ();
+pub type GetCollection = Vec<GetCollectionItem>;
+pub type DeleteCollection = Option<DeleteBsos>;
+pub type DeleteBso = ();
+pub type PutBso = u64;
+
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(untagged)]
+pub enum GetCollectionItem {
+    Id(String),
+    Bso(GetBso),
+}
+
+#[derive(Debug, Default, Deserialize, Serialize)]
+pub struct GetBso {
+    pub id: String,
```

Comment:
The old code was also returning `collection_id` here, but I don't think we ever need to expose those ids externally since all the endpoints take a collection name instead, right?

Reply:
yup